src/main/java/io/vertx/core/http/impl/Http1xServerResponse.java

```diff
@@ -745,14 +744,14 @@ private Http1xServerResponse write(ByteBuf chunk, PromiseInternal<Void> promise)
         }
       }
       bytesWritten += chunk.readableBytes();
-      HttpObject msg;
       if (!headWritten) {
         prepareHeaders(-1);
-        msg = new AssembledHttpResponse(head, version, status, headers, chunk);
+        conn.writeToChannel(new VertxHttpResponse(head, version, status, headers));
```

Comment:
this is the sole case I'm worried about @vietj

Reply:
could Netty have a similar default type ?

Reply:
With the `head` element in?

Reply:
with the headers+chunk